lang/janet/Portfile

```diff
@@ -23,21 +23,46 @@ long_description    Janet is a functional and imperative programming \
 
 homepage            https://janet-lang.org/
 
-checksums           rmd160  1d77318cc48b524ee50a398c9e0a36bedac03cda \
-                    sha256  b65a83f27ec80b1db8e046aed6ac3eb5ed92d843bdbdc5f4d279f10664862de6 \
-                    size    601152
+checksums           rmd160  7e02e3db9be2b43252339eef553a9a52d586e341 \
+                    sha256  ce07c705aea3ec751ba7db3385f8dc0ddfad75a6ff1f86dc525476e977a8e4e1 \
+                    size    494150
 
 # Fixes passing correct link flags to boot_strap janet
-# and incorrect use of environ on MacOS
+# and incorrect use of environ on macOS
 patch.pre_args      -p1
 
 # Fixes linking of boot strap binary on older systems
 patchfiles-append   fix-janet_boot-link-legacysupport.diff
 
-# Correct use of NSGetEnviron() on Apple systems
-# https://github.com/janet-lang/janet/pull/688
-patchfiles-append   use-nsgetenviron-on-apple.diff
-
 destroot.keepdirs   ${destroot}${prefix}/lib/${name}
 
 test.run            yes
+
+notes "
+Janet has a package ecosystem (similar to how Nodejs does) and\
+these packages are managed by the package manager jpm. However\
+jpm is not installed by default. You have a few options which\
+are not mutually exclusive.
+
+(1) Use jpm packages managed by MacPorts. These are available\
+    with prefix 'janet-' so the package 'joy' is on MacPorts\
+    as 'janet-joy'.
+
+    e.g.
+        \$ port install janet-joy
+
+(2) Use jpm packages via jpm itself. It is _not_ recommended\
+    to install packages globally as they would be unmanaged by\
+    MacPorts, however, jpm has functionality to install packages\
+    at a custom directory (e.g. local to your janet project).\
+    This is configured via JANET_TREE for jpm. See the\
+    installation notes for jpm for more information.
+
+    e.g.
+        \$ port install jpm
+        \$ JANET_TREE ~/projects jpm install joy
```

Comment:
Just a drive-by comment, but this should probably be `\$ JANET_TREE=~/projects jpm install joy`.

Reply:
Hehe missed the cheeky equals, thanks